t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -197,9 +197,18 @@ sub _default_check_interval {
 
 sub wait_for_ajax {
     my $check_interval = _default_check_interval(shift);
-    while (!$_driver->execute_script("return jQuery.active == 0")) {
+    my $timeout = 60 * 5;
+
+    while (!$_driver->execute_script("return typeof jQuery != 'undefined' && jQuery.active == 0")) {
```

Comment:
The useless use of typeof and the `!=` operator instead of `!==` is not nice. I'd simply use `window.jQuery && jQuery.active === 0`.